apps/wallet/src/ui/app/hooks/useOwnedNFT.ts

```diff
@@ -15,7 +15,7 @@ export function useOwnedNFT(
     nftObjectId: string | null,
     address: SuiAddress | null
 ) {
-    const data = useGetObject(nftObjectId!);
+    const data = useGetObject(nftObjectId || '');
```

Comment:
This hook doesn't look like it handles nullable object IDs, instead of this defaulting we should update it to handle this properly